doc/advanced.rst

```diff
@@ -277,29 +277,53 @@ filter: ``('a', 'b', 'foo')``.
 Deprecated Filters
 ~~~~~~~~~~~~~~~~~~
 
-You can mark a filter as being deprecated by setting the ``deprecated`` option
-to ``true``. You can also give an alternative filter that replaces the
-deprecated one when that makes sense::
+.. versionadded:: 3.14
+
+    The ``deprecated_info`` option was added in Twig 3.14.
```

Comment:
maybe `deprecation_info` would be a better name for that, as `deprecated_info` reads like the info itself is deprecated (while it is the info about a deprecation)

Reply:
Makes sense, renamed.